packages/x-data-grid/src/models/api/gridCoreApi.ts

```diff
@@ -69,9 +69,17 @@ export interface GridCorePrivateApi<
    */
   mainElementRef: React.RefObject<HTMLDivElement>;
   /**
-   * The React ref of the grid virtual scroller container element.
+   * The React ref of the grid's virtual scroller container element.
    */
   virtualScrollerRef: React.RefObject<HTMLDivElement>;
+  /**
+   * The React ref of the grid's vertical virtual scrollbar container element.
+   */
+  virtualScrollbarVerticalRef: React.RefObject<HTMLDivElement>;
+  /**
+   * The React ref of the grid's horizontal virtual scrollbar container element.
+   */
+  virtualScrollbarHorizontalRef: React.RefObject<HTMLDivElement>;
```

Comment:
Do these need to be public? I'd rather avoid exposing stuff until it needs to be exposed, once it's public it can't be refactored/removed.

Reply:
I also wanted to refactor the names to remove the "virtual" prefix from everywhere, it would make for more readable names and shorter lines. It could be interesting to name those just `scrollbarXxxxRef`.

Reply:
They're defined in the `GridCorePrivateApi` interface, so not public 👍🏻

Reply:
The only public thing around this are few classes
https://mui.com/x/api/data-grid/data-grid/#data-grid-classes-MuiDataGrid-virtualScroller

we can make the change in v8 if you want to have a cleanup